backend/recipes/base.py

```diff
@@ -77,85 +76,77 @@
                             The parameters to pass to the task function. This
                             is a dictionary that will be passed as the single
                             parameter to this function.
-                done
-                    Always set to true and signals that the recipe was
-                    successfully completed.
+                is_final
+                    When True - signals that this is recipe's final step
```

Comment:
I'm OK with this change, but it will require updates to the `recipe-press` repo as well. [For example, in this file](https://github.com/FourThievesVinegar/recipe-press/blob/master/src/contexts/RecipeContext.tsx).

if it's just renaming the key, might be better to hold off and make recipe changes their own set of PRs across both repos.

Reply:
reverted